src/Microsoft.NET.Sdk.Razor/Microsoft.NET.Sdk.Razor.csproj

```diff
@@ -1,30 +1,23 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <Description>Razor is a markup syntax for adding server-side logic to web pages. This package contains MSBuild support for Razor.</Description>
-    <TargetFrameworks>netstandard2.0;net46</TargetFrameworks>
+    <TargetFrameworks>netcoreapp2.2;netstandard2.0;net46</TargetFrameworks>
```

Comment:
hmmm?

Reply:
Required to reference Razor.Tools. Once we reference it we can make the build order more reliable and avoid file contention.